qutip/tests/core/data/test_norm.py

```diff
@@ -0,0 +1,86 @@
+from . import test_mathematics as testing
+import numpy as np
+from scipy import linalg
```

Comment:
```suggestion
import scipy
```
Suggesting making this change now so we don't have to tidy it up later.

Reply:
scipy's components are separate packages which aren't auto-imported, so it needs to be `import scipy.linalg`

Reply:
This works accidentally because `import qutip` does `import scipy.linalg` somewhere.